content/concepts/kata/translations.md

```diff
@@ -5,73 +5,124 @@ sidebar: explanation-kata
+- **Verifying** if the translation holds to [translation authoring guidelines (TODO)]().
 
-## Aproving a translation
+After a review is completed, the reviewer can finalize it with one of following actions:
 
-TBD
+- **Leaving a comment** with remarks, suggestions and issues which should be addressed by the translator.
+- **Fixing problems found during the review**. The translation fork may be edited directly only by its author. If a user wants to make changes to a translation fork created by another user, they have to fork it first, edit the code, and publish the changes as a new fork.
+- **Approving the translation**, which introduces a new language to the kata available for solving.
+- **Rejecting the translation** if it cannot be fixed.
 
-- who can approve
-- Approver is marked as coauthor
+### Approving a translation
+
+Pending translations can be approved by either the kata author or any user with the [privilege to coauthor](/references/gamification/privileges/) apart from the translator himself/herself, under the following conditions:
+
+- translation is older than a week, or
+- kata author has been inactive for over a month.
+
+When the translation is approved, a new language is immediately available for solving. There's also a honor reward, but remember that it's granted to the author of the **initial fork** and not that of the accepted fork!
+
+The user who approved the translation should also go through the kata discourse and mark suggestions posted by the translator as resolved.
+
+**NOTE:** at the time of writing, a user who _approves_ a translation is marked as a **kata contributor**. Users with the "contributor" flag seem to be affected by a bug preventing them from earning any future points they would be rewarded for authoring activities on kata they contributed to. Be aware that as a result, approving a translation will render you ineligible for honor points you would otherwise earn by translating, editing or fixing the kata in the future unless the bug is fixed.
 
 ### Merge Issues
 
-Oftentimes if two translations are created around the same time, they may both update the description. When approving the 2nd translation, the 1st approved translation will have already updated the description. This will cause a merge conflict which will need to be resolved. You can do this by forking the 2nd translation and updating the description. `<<<<<<<<`, `=========` and `>>>>>>>>` merge labels will show up in the description to help identify where the merge issues are.
+Oftentimes if two translations are created around the same time, they may both update the description. When approving the 2nd translation, the 1st approved translation will have already updated the description. This will cause a merge conflict which has to be resolved. You can do this by forking the 2nd translation and updating the description. `<<<<<<<<`, `=========` and `>>>>>>>>` merge labels will show up in the description to help identify where the merge issues are.
```

Comment:
```suggestion
Oftentimes if two translations are created around the same time, they may both update the description. When approving the 2nd translation, the 1st approved translation will already have updated the description. This will cause a `merge conflict` which has to be resolved.
You can do this by forking the 2nd translation and updating the description. `<<<<<<<<`, `=========` and `>>>>>>>>` merge labels will show up in the description to help identify where the merge issues are. Another solution is to start a new translation to grab the last version of the kata description, then copy/paste it in the translation to modify.
```